machine/wnc_x86_64/INSTALL

```diff
@@ -0,0 +1,36 @@
+=============================================
+Installing ONIE on WNC x86_64
+=============================================
+
+Cross-Compiling ONIE
+====================
+
+Change directories to ``build-config`` to compile ONIE.
+
+To compile ONIE first change directories to ``machine/wnc_x86_64`` and then
+type ``"./wnc_onie_build.sh"``.  For example::
+
+  $ cd machine/wnc_x86_64
+  $ ./wnc_onie_build.sh
```

Comment:
Adding custom build scripts into the repo is not acceptable.
You can maintain custom build scripts on your end.